app/javascript/components/server-components/Settings/PaymentsPage.tsx

```diff
@@ -888,9 +889,27 @@ const PaymentsPage = (props: Props) => {
                 options={PAYOUT_FREQUENCIES.map((frequency) => ({
                   id: frequency,
                   label: frequency.charAt(0).toUpperCase() + frequency.slice(1),
+                  disabled: frequency === "daily" && !props.payout_frequency_daily_supported,
```

Comment:
Per a conversation with Sahil, we will always show "Daily" as an option, but disable it when users are not supported (along with a note about eligibility in the small text)